tests/vmi_lifecycle_test.go

```diff
@@ -135,6 +135,15 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			tests.WaitForSuccessfulVMIStart(vmi)
 		})
 
+		It("[test_id:6095]should start in paused state if start strategy set to paused", func() {
+			strategy := v1.StartStrategyPaused
+			vmi.Spec.StartStrategy = &strategy
+			vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+			Expect(err).To(BeNil(), "Create VMI successfully")
+			tests.WaitForSuccessfulVMIStart(vmi)
+			tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstancePaused, 30)
```

Comment:
can you add an "unpause" here to this test condition as well? I know we already test unpause, i'd just like to verify we can enter that state after starting the VMI as paused.

Reply:
Done. This was also tested manually.